### ISSUE TITLE
Talon Strike not working

### ISSUE THREAD

**Issue by Psychadelic** [Valhalla-Gaming/Tracker#8240]
**Describe the issue you're having**:

Talon Strike talent on Survival hunter artifact weapons is not working. I tested.
**Explain how you expect it work**:

Your basic attacks have a chance to trigger two rapid additional blows. -  Thats what the talent says.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Links to Wowhead, YouTube, etc**:

**Comment by Bloodshade** [comment]
https://github.com/Valhalla-Gaming/Tracker/issues/8293

**Comment by ixodan** [comment]
yes its bugged but from the op side. it has 100% chance procs always .

**Comment by Proklet** [comment]
It works as intended now. At least for me, every three or four auto-attacks it triggers.
